src/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -39,7 +39,7 @@
 
 namespace System.Globalization
 {
-    public partial class CultureInfo : IFormatProvider
+    public partial class CultureInfo : IFormatProvider, ICloneable
```

Comment:
We are using fork of this globalization implementation in CoreCLR on Unix. The same changes should be made under `src\mscorlib\corefx` in coreclr repo.